.github/workflows/pr-validation.yml

```diff
@@ -5,9 +5,15 @@ name: "Validate Pull Request"
 
 on:
   push:
-    branches: [ main, dev ]
+    branches:
+      - main
+      - dev
+      - feat/[0-9]+.[0-9]+.[0-9]+ # Major rev branches
```

Comment:
We could change it to `feat/[2-9]+.[0-9]+.[0-9]+` since we know we won't or shouldn't ever use 0.x.x or 1.x.x anymore. Comment applicable in a few other places.

Reply:
Love it!